src/libproc/sys/macos.rs

```diff
@@ -61,7 +61,7 @@ pub(crate) fn listpids(proc_type: ProcFilter) -> io::Result<Vec<u32>> {
     match ret {
         value if value <= 0 => Err(io::Error::last_os_error()),
         _ => {
-            let items_count = ret as usize / mem::size_of::<u32>() - 1;
```

Comment:
@mjpieters I think you contributed this code originally. 

Do you know where the -1 came from?